src/app/login/login.component.html

```diff
@@ -1,7 +1,7 @@
-<button *ngIf="!user" (click)="login()" mat-flat-button>login</button>
+<button color='warn' *ngIf="!user" (click)="login()" mat-flat-button>login</button>
```

Comment:
The three selectors of color are `warn`, `accent`, and `primary`. We can edit these.